packages/fxa-auth-server/lib/senders/templates/multiSubscriptionsPaymentExpired.html

```diff
@@ -0,0 +1,19 @@
+{{#*inline "title"}}
+  {{t "Your credit card is about to expire" }}
+{{/inline}}
+
+{{#*inline "content"}}
+  {{t "The credit card you're using to make payments for the following subscriptions is about to expire." }}
+  <br><br>
+  <ul>
+    {{#subscriptionProductNames}}
+    <li>{{t . }}</li>
```

Comment:
Same thing here - can just use `{{ }}` since there's no content to extract for l10n